src/features/ui/WrappedToggleInput/WrappedToggleInput.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react';
+
+import { ToggleGroup, ToggleOptions } from '@zero-tech/zui/components';
+import { Wrapper } from '../Wrapper/Wrapper';
+
+import styles from './WrappedToggleInput.module.scss';
+
+export type WrappedToggleInputProps = {
+	className?: string;
+	options: ToggleOptions;
+	label: string;
+	value: string;
+	info: string;
+	learnMoreHref?: string;
```

Comment:
I feel `href` would be better than `learnMoreHref`. Just in case design ever changed the text content of the link. Let me know thoughts

Reply:
Agree! Will update.